src/recordlinker/routes/algorithm_router.py

```diff
@@ -0,0 +1,102 @@
+"""
+recordlinker.routes.algorithm_router
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This module implements the algorithm router for the RecordLinker API. Exposing
+the algorithm configuration API endpoints.
+"""
+
+import typing
+
+import fastapi
+import sqlalchemy.orm as orm
+
+from recordlinker import schemas
+from recordlinker.database import get_session
+from recordlinker.linking import algorithm_service as service
+
+router = fastapi.APIRouter()
+
+
+@router.get("", status_code=fastapi.status.HTTP_200_OK)
```

Comment:
Should we do `@router.get("/", status_code=fastapi.status.HTTP_200_OK)` just to maintain convention here?

Reply:
Correct me if I'm wrong, but the existing convention is no trailing slashes right?

So the current DIBBS API endpoint is, `/link-record` and not `/link-record/`.

Reply:
True, I was basing it off `@app.get("/")` endpoint in `main.py`. But you're right, all the other endpoints don't have trailing slashes.

Reply:
Yeah, that's true for the health endpoint, but I think that's just a necessity because it's the root URL.

Note that I'm using a [prefix when initializing the router](https://github.com/CDCgov/RecordLinker/blob/feature/65-algorithm-config/src/recordlinker/main.py#L34).  So for this endpoint the URL will be "/algorithm".  If we add a slash here, we'll have "/algorithm/".

Reply:
Gotcha, I overlooked that there's already `/algorithm`. Looks good!